### PR TITLE
[openwebnet] Thermo: new channels and README updates

### LINKED ISSUE #12019: [openwebnet] Thermo: add heating channel of type switch in addition to heatingValves
This feature was available for OH2 but not currently supported in OH3. It works at start up but the valve items are not updated after start up.

It is useful to create a heat map chart to track the whole house heating demand zone by zone

image

### PR DESCRIPTION
**Pull request by mvalla**
Depends on #16626.

This PR adds these changes:
- new channels: `targetTemperature`, `heating` and `coolng` for `bus_thermo_zone` Things
- code for `updateModeAndFunction()` improved/fixed for better handling of  `*4*1*where##` messages when changing between Winter/Summer thermo functions
- updated README to add more references to MyHOME system name; added new device images.

Fixes #12019.

### CODE REVIEW

**Comment by lolodomo** [comment]
Rebase needed to solve conflicts.

**Comment by mvalla** [comment]
> Rebase needed to solve conflicts.

@lolodomo done thanks!

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml [hunk 1]
```diff
@@ -113,6 +113,16 @@
 		<state pattern="%.1f %unit%" step="0.5"/>
 	</channel-type>
 
+	<channel-type id="targetTemperature" advanced="true">
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:116]: What is the difference between setpointTemperature and targetTemperature ?

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml [hunk 2]
```diff
@@ -113,6 +113,16 @@
...
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Target temperature (read only)</description>
+		<tags>
+			<tag>Temperature</tag>
+		</tags>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:122]: Something missing. Is it a measurement, a setpoint ??

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:116]: - `setpointTemperature` -> The zone or Central Unit manual setpoint temperature
- `targetTemperature` ->  The current zone target temperature according to `mode`, `setpointTemperature` and `localOffset`.

For example one thermostat can have the `setpointTemperature` set to 22° ad the `locaIOffset` (physical knob) set to "-1" so `targetTemperature` would be 21°.
Another example is `setpointTemperature`=22° but thermostat is locally set to "anti-freeze", so `targetTemperature` would be 7°.

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:122]: in fact it's neither. See previus answser. If one has to be chosen then setpoint is better.

Comment (lolodomo) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:122]: In that case, please set no tag at all

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml:116]: Understood.

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml (hunk 2 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml]: done

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**